go.mod

```diff
@@ -7,11 +7,11 @@ require (
 	github.com/filecoin-project/go-address v0.0.5
 	github.com/filecoin-project/go-cbor-util v0.0.0-20191219014500-08c40a1e63a2
 	github.com/filecoin-project/go-commp-utils v0.1.1-0.20210427191551-70bf140d31c7
-	github.com/filecoin-project/go-data-transfer v1.7.6
+	github.com/filecoin-project/go-data-transfer v1.7.8-0.20210827051445-c354689456a5
```

Comment:
Flagging that this should change to go-data-transfer v1.8.0 when the corresponding PR is merged and released upstream.